### PR TITLE
Features test

### PR DESCRIPTION
**Pull request by GuilhermeGiachellin**
## What was done in this project

- Created tests for the components.
- Created tests for the logic.
- The tests were made using pair programming.

### CODE REVIEW

File: src/logic/logicTests/calculate.test.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
+import calculate from '../calculate.js';
+
```
Comment (bellom) [inline src/logic/logicTests/calculate.test.js:2]: We need to make sure all the functionalities & scenarios of the operate.js and calculate.js modules are covered using Jest.

So we need to write a unit test for `.` operator functionality in `calculate.js`, when `buttonName === .`.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @GuilhermeGiachellin  👏 , 

### Awesome job so far!! :+1: 

#### You have done wonderfully well with the unit tests, setting up linter and good commit messages 🔥, However, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Required Changes! ♻️ 

_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/logic/logicTests/calculate.test.js (hunk 1 above)
Comment (afcastaneda223) [reply src/logic/logicTests/calculate.test.js:2]: add test for dot operator in calculate.test.js

thank you

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @GuilhermeGiachellin &  @afcastaneda223, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**